tests/Backend/CommonPart1/DeleteRowsTest.php

```diff
@@ -367,52 +367,183 @@ public function tableDeleteRowsByFiltersData(): array
                     ],
                 ],
             ],
-            'where filter: valuesByTableInStorage' => [
-                [
-                    'whereFilters' => [
-                        [
-                            'column' => 'city',
-                            'valuesByTableInStorage' => [
-                                    'tableId' => 'table',
-                                    'column' => 'city',
-                            ],
+        ];
+    }
+
+    public function testDeleteByValuesInWorkspaceWithInvalidData(): void
+    {
+        $importFile = __DIR__ . '/../../_data/users.csv';
+        $tableId = $this->_client->createTableAsync($this->getTestBucketId(self::STAGE_IN), 'users', new CsvFile($importFile));
+
+        $workspaces = new Workspaces($this->_client);
+
+        foreach ($workspaces->listWorkspaces() as $workspace) {
+            $workspaces->deleteWorkspace($workspace['id'], [], true);
+        }
+
+        $runId = $this->_client->generateRunId();
+        $this->_client->setRunId($runId);
+        $this->_client->setRunId($runId);
+
+        $workspace = $workspaces->createWorkspace();
+        $backend = WorkspaceBackendFactory::createWorkspaceBackend($workspace);
+        $backend->createTable('USERS', ['ID' => 'INT', 'name' => 'VARCHAR(255)']);
+
+        $backend->executeQuery("INSERT INTO USERS VALUES (1, 'martin');");
+        $backend->executeQuery("INSERT INTO USERS VALUES (3, 'ondra');");
+
+        // test invalid table
+        try {
+            $this->_client->deleteTableRows($tableId, [
+                'whereFilters' => [
+                    [
+                        'column' => 'id',
+                        'valuesByTableInWorkspace' => [
+                            'workspaceId' => $workspace['id'],
+                            'table' => 'NOTEXISTING',
+                            'column' => 'ID',
                         ],
                     ],
                 ],
-                // no rows should be deleted because valuesByTableInStorage doesn't do anything yet
```

Comment:
todle testovalo jen existenci tech parametru a jejich validaci, ted uz to zacina mazat